content/en/docs/concepts/architecture/cri.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Container Runtime Interface (CRI)
+content_type: concept
+weight: 10
```

Comment:
I'd make the weight higher (de-emphasizing the page).

Reply:
Changed it to `50`.